specification/transport_layer/can.tex

```diff
@@ -261,23 +278,28 @@ \subsubsection{Payload discriminator}\label{sec:can_payload_discriminator}
 for different data field values.
 A possible way of initializing the payload discriminator value is to apply the transfer CRC function
 (as defined in the section \ref{sec:transfer_crc})
-to the contents of the anonymous message, and then use any 10 bits of the result.
+to the contents of the anonymous message, and then use the seven least significant bits of the result.
 Nodes that adopt this approach will be using the same payload discriminator value for identical messages,
 which is acceptable since this will not trigger an error on the bus.
 
-Since the discriminator is only 10 bits long,
-the probability of having multiple nodes emitting CAN frames with the same CAN ID
-but different data can exceed 0.1\%, which is significant.
+Since the discriminator is only seven bits long (128 possible values),
+a collision where multiple nodes emit CAN frames with different data but same CAN ID is likely to happen
+despite the randomization measures described earlier.
 Therefore, the protocol must account for possible errors on the CAN bus triggered by CAN ID collisions.
 In order to comply with this requirement,
 UAVCAN requires all nodes to immediately abort transmission of all anonymous transfers once an error on
```

Comment:
Maybe include "If the transport protocol use automatic retransmission, this must be disabled for anonymous frames"

Reply:
Added.